pkg/importer/dataStream_test.go

```diff
@@ -300,7 +300,7 @@ func createTestData() map[string]string {
 }
 
 func Test_dataStream_constructReaders(t *testing.T) {
-	imageDir, _ := filepath.Abs(testImagesDir)
+	imageDir, _ := filepath.Abs(TestImagesDir)
```

Comment:
you'll want to look at the changes to this func.  Since the filename has changed I'm not sure how it will get merged. Bottom line is I fixed a bug where `constructReaders` was called twice. I changed the test to verify number of readers instead of err.

Reply:
This PR does not change code other than import paths (here it removes the package prefix since it's now in the same package).  There should be no issues with changes other PRs have already made to this test.